eng/pipelines/patch_release_client.txt

```diff
@@ -23,6 +23,7 @@ com.azure:azure-ai-formrecognizer # Tests owner: samvaity
 com.azure:azure-ai-metricsadvisor # Tests owner: samvaity
 com.azure:azure-ai-textanalytics # Tests owner: mssfang
 com.azure:azure-communication-callautomation # Tests owner: minwoolee-msft, juntuchen-msft
+com.azure:azure-communication-jobrouter # Tests owner: cparisineti
```

Comment:
Do you mind sort the order that moving this line after azure-communication-identity?

Reply:
Done.